Source/WTF/wtf/posix/FileSystemPOSIX.cpp

```diff
@@ -270,15 +271,17 @@ std::pair<String, PlatformFileHandle> openTemporaryFile(StringView prefix, Strin
     // This is OK for now since the code using it is built on macOS only.
```

Comment:
The comment above saying we can’t implement the suffix is incorrect, because we could do it by using `mkostemps` instead of `mkostemp`.